cmd/neutree-cli/app/cmd/launch/manifests/core.go

```diff
@@ -0,0 +1,197 @@
+package manifests
+
+import (
+	"embed"
+)
+
+//go:embed db.tar
+var NeutreeCoreDBInitScripts embed.FS
+
+var NeutreePrometheusScrapeConfig = `
+global:
+  scrape_interval: 30s # Set the scrape interval to every 30 seconds. Default is every 1 minute.
+
+scrape_configs:
+# Scrape from each Ray node as defined in the service_discovery.json provided by Ray.
+- job_name: 'neutree'
+  file_sd_configs:
+  - files:
+    - '/etc/prometheus/scrape/*.json'
+`
+
+//nolint:lll
+var NeutreeCoreDockerComposeManifests = `
```

Comment:
Shall we read the docker-compose.yml from FS?

Reply:
This will look better, and I will unify the reading method.